packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/api/index.ts

```diff
@@ -0,0 +1,149 @@
+  getServerMetricsConfig,
+  getTest,
+  getTestFile,
+  listTestFiles,
+  listTests,
+  uploadTestFile,
+  deleteTestFile,
+  deleteTest,
+  CreateOrUpdateTestOptions,
+  CreateOrUpdateAppComponentsOptions,
+  CreateOrUpdateServerMetricsConfigOptions,
+  GetAppComponentsOptions,
+  GetServerMetricsConfigOptions,
+  GetTestOptions,
+  GetTestFileOptions,
+  ListTestFilesOptions,
+  ListTestsOptions,
+  UploadTestFileOptions,
+  DeleteTestFileOptions,
+  DeleteTestOptions,
+} from "./loadTestAdministration/index.js";
+export {
+  createLoadTestRun,
+  LoadTestRunClientOptions,
+  AzureLoadTestingContext as LoadTestRunClientAzureLoadTestingContext,
+  Test as LoadTestRunClientTest,
+  PassFailCriteria as LoadTestRunClientPassFailCriteria,
+  PassFailMetric as LoadTestRunClientPassFailMetric,
+  PFMetrics as LoadTestRunClientPFMetrics,
+  PFAgFunc as LoadTestRunClientPFAgFunc,
+  PFAction as LoadTestRunClientPFAction,
+  PFResult as LoadTestRunClientPFResult,
+  Secret as LoadTestRunClientSecret,
+  SecretType as LoadTestRunClientSecretType,
+  CertificateMetadata as LoadTestRunClientCertificateMetadata,
+  CertificateType as LoadTestRunClientCertificateType,
+  LoadTestConfiguration as LoadTestRunClientLoadTestConfiguration,
+  OptionalLoadTestConfig as LoadTestRunClientOptionalLoadTestConfig,
+  TestInputArtifacts as LoadTestRunClientTestInputArtifacts,
+  FileInfo as LoadTestRunClientFileInfo,
+  FileType as LoadTestRunClientFileType,
+  FileStatus as LoadTestRunClientFileStatus,
+  AppComponent as LoadTestRunClientAppComponent,
+  ResourceMetric as LoadTestRunClientResourceMetric,
+  TestRun as LoadTestRunClientTestRun,
+  ErrorDetails as LoadTestRunClientErrorDetails,
+  TestRunStatistics as LoadTestRunClientTestRunStatistics,
+  TestRunArtifacts as LoadTestRunClientTestRunArtifacts,
+  TestRunInputArtifacts as LoadTestRunClientTestRunInputArtifacts,
+  TestRunOutputArtifacts as LoadTestRunClientTestRunOutputArtifacts,
+  PFTestResult as LoadTestRunClientPFTestResult,
+  Status as LoadTestRunClientStatus,
+  MetricDefinition as LoadTestRunClientMetricDefinition,
+  NameAndDesc as LoadTestRunClientNameAndDesc,
+  AggregationType as LoadTestRunClientAggregationType,
+  MetricUnit as LoadTestRunClientMetricUnit,
+  MetricAvailability as LoadTestRunClientMetricAvailability,
+  TimeGrain as LoadTestRunClientTimeGrain,
+  MetricNamespace as LoadTestRunClientMetricNamespace,
+  DimensionFilter as LoadTestRunClientDimensionFilter,
+  TimeSeriesElement as LoadTestRunClientTimeSeriesElement,
+  MetricValue as LoadTestRunClientMetricValue,
+  DimensionValue as LoadTestRunClientDimensionValue,
+  Interval as LoadTestRunClientInterval,
+  DimensionValueList as LoadTestRunClientDimensionValueList,
+  testRun,
+  createOrUpdateAppComponents as LoadTestRunClientcreateOrUpdateAppComponents,
+  createOrUpdateServerMetricsConfig as LoadTestRunClientcreateOrUpdateServerMetricsConfig,
+  deleteTestRun,
+  getAppComponents as LoadTestRunClientgetAppComponents,
```

Comment:
There's operation name conflicts in the api layer. In RLC layer we use operation group as the prefix, wonder what's the solution in API layer. 
Probably need to check out the operation name in client.tsp